add documentation and change log entry for the feature toggle UI (#4447)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-36-0-feature-toggle-ui.md

---
date: '2026-02-11'

> title: Add feature toggle view to Administration app

Reviewer: ```suggestion
title: Added feature toggle view to the Administration application
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: ''
version: 1023.36.0
environment_availability:
  - label: eu-latest.cumulocity.com
    date: '2026-02-11'
  - label: apj.cumulocity.com
    date: '2026-02-12'
  - label: jp.cumulocity.com
    date: '2026-02-12'
  - label: emea.cumulocity.com
    date: '2026-02-13'
  - label: us.cumulocity.com
    date: '2026-02-13'
  - label: cumulocity.com
    date: '2026-02-13'
---

> A new view was added to the Administration application listing the state of the currently available feature toggles within the tenant.

Reviewer: ```suggestion
A new view has been added to the Administration application that lists all currently available feature toggles within the tenant and shows their state (enabled/disabled).
```

> It allows you to change the state of feature toggles and reset them to their default state.

Reviewer: ```suggestion
Moreover, you can change the state of feature toggles from here and reset them to their default state.

For details, see [Feature toggles](/standard-tenant/changing-settings/#feature-toggles).
```
